public/less/tweaks.less

```diff
@@ -17,6 +17,27 @@ header.navbar.navbar-inverse {
     background-color: @brand-primary;
 }
 
+.logo{
+    padding-top: @padding-large-vertical;
+    padding-bottom: @padding-large-vertical;
+}
+.homepage h2{
+    font-weight:300;
+    font-weight:20px;
```

Comment:
Also is this supposed to be `font-size`?  And if so, shouldn't that be `@font-size-xxx` variable?  (Where xxx is whatever is appropriate)